inc/define.php

```diff
@@ -291,6 +291,10 @@
                                                   'DeviceCase', 'DevicePowerSupply', 'DeviceGeneric',
                                                   'DeviceSimcard', 'DeviceSensor'];
 
+if (version_compare(GLPI_VERSION, '9.5.1', '>')) {
```

Comment:
Maybe check if class exists rather than compare to version

Reply:
done